### DISCUSSION TITLE
Some questions about "GC-MS EI Data Analysis"

### DISCUSSION THREAD

**Discussion by TritAlpha** [CCMS-UCSD/GNPS_Workflows#842] [Q&A]
when i use "Data Processing - Deconvolution” function , 
i receive  "The workflow uses MSHub algorithm that conducts dataset-wide deconvolution and alignment of GC-MS data”. This workflow requires a recommended minimum of 10 samples to function"
but i only have one GCMS sample , i still input one sample data , unfortunately the workflow is failed. So i came here for help, i want to know how to build a molecular network if there is only one GCMS sample. Looking forward to the help of enthusiastic people。

**Comment by mwang87** [comment]
I'd recommend using a different tool for deconvolution. Here is a guide on using MZMine:

https://ccms-ucsd.github.io/GNPSDocumentation/gc-ms-deconvolution/#gc-ms-data-processing-with-adap-mzmine

Best,

Ming

**Comment by TritAlpha** [comment]
您的信件我已经收到。 This is an automatic reply, confirming that your e-mail was received.Thank you
